Makefile

```diff
@@ -752,9 +751,19 @@ devbox-grpc:
 		lib/multiplexer/test/ping.proto \
 		lib/web/envelope.proto
 
+	# proto files within the api module must be passed with the 'M' flag. This way
+	# protoc will generate files with the correct import paths in the case where
+	# the import path has a version suffix, e.g. github.com/gravitational/teleport/api/v8
+	$(eval API_IMPORT_PATH=$(shell go run build.assets/go-modules/print-api-import-path/main.go))
```

Comment:
Could we drop the eval and turn API_IMPORT_PATH into a make variable?

(Don't forget to drop the double `$$`s below after you do it.)

Reply:
Won't that require us to compile and run a Go program on ever invocation of make, even if this target isn't called?